packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -77,7 +77,7 @@ export const MenuTitleActionsUI = styled.span`
 
 	.components-button.is-small {
 		color: ${ G2.lightGray.ui };
-		margin-right: 2px; // Avoid hiding the focus outline
+		margin-right: ${ space( 0.25 ) }; // Avoid hiding the focus outline
```

Comment:
I wonder if we should use `space` for rules specifically needed to work around visual glitches (this one here, but also the paddings to "leave room for icons/buttons" at lines 101-102).

There's a chance I'm just irked by the decimal, though.
What about rounding up? I don't think it would affect stuff too much, but feel free to disregard this suggestion if it looks off to you. 👍 
- `space( 0.25 )` -> `space( 0.5 )` (2 -> 4)
- `space( 3.75 )` -> `space( 4 )` (30 -> 32)

Reply:
> There's a chance I'm just irked by the decimal, though.
What about rounding up? I don't think it would affect stuff too much, but feel free to disregard this suggestion if it looks off to you. 👍

I'll give it a try now 👍

Reply:
| space( 3.75 ) | space( 4 ) |
| -- | -- |
| <img width="302" alt="Screen Shot 2020-10-28 at 9 27 18 AM" src="https://user-images.githubusercontent.com/5414230/97466270-25cfc580-1900-11eb-98e0-9db4b843f673.png"> | <img width="303" alt="Screen Shot 2020-10-28 at 9 26 40 AM" src="https://user-images.githubusercontent.com/5414230/97466327-36803b80-1900-11eb-9f5d-8b3870be0bb5.png"> |

| space( 0.25 ) | space( 0.5 ) |
| -- | -- |
|  | <img width="303" alt="Screen Shot 2020-10-28 at 9 26 40 AM" src="https://user-images.githubusercontent.com/5414230/97466327-36803b80-1900-11eb-9f5d-8b3870be0bb5.png"> |

Reply:
| space( 3.75 ) | space( 4 ) |
| -- | -- |
|<img width="302" alt="Screen Shot 2020-10-28 at 9 27 18 AM" src="https://user-images.githubusercontent.com/5414230/97468697-ce7f2480-1902-11eb-9c5d-f6d6304a0ad3.png">|<img width="303" alt="Screen Shot 2020-10-28 at 9 26 40 AM" src="https://user-images.githubusercontent.com/5414230/97468714-d2ab4200-1902-11eb-9140-ab90af904b20.png">|

| space( 0.25 ) | space( 0.5 ) |
| -- | -- |
|<img width="301" alt="Screen Shot 2020-10-28 at 9 51 06 AM" src="https://user-images.githubusercontent.com/5414230/97469118-43eaf500-1903-11eb-98e1-948a1341d6ff.png">|<img width="301" alt="Screen Shot 2020-10-28 at 9 50 05 AM" src="https://user-images.githubusercontent.com/5414230/97469141-4b120300-1903-11eb-8293-7ddaa96a0b19.png">|

Reply:
> What about rounding up? I don't think it would affect stuff too much, but feel free to disregard this suggestion if it looks off to you. 

The changes are almost indistinguishable, so I'll go ahead and round up the decimals.